src/SmartComponents/AddSystemModal/AddSystemModal.js

```diff
@@ -96,7 +97,15 @@ export class AddSystemModal extends Component {
                         <Button
                             key="confirm"
                             variant="primary"
-                            onClick={ this.confirmModal }>
+                            onClick={ this.confirmModal }
+                            isDisabled={
+                                (entities && entities.selectedSystemIds && entities.selectedSystemIds.length === 0) &&
+                                selectedBaselineIds.length === 0 &&
+                                selectedHSPIds.length === 0
+                                    ? true
+                                    : false
```

Comment:
Given the condition of the ternary operator is a boolean, you can drop the ternary all together and simply pass the result of the expression to `isDisabled`.